### PR TITLE
Add note to image regarding invalid references

### PR DESCRIPTION
**Pull request by jacobkeeler**

### CODE REVIEW

File: docs/Common/Structs/index.md [hunk 1]
```diff
@@ -427,9 +427,9 @@
 
 |Name|Type|Mandatory|Additional|Description|
 |:---|:---|:--------|:---------|:----------|
-|value|String|true|maxlength: 65535||
-|imageType|Common.ImageType|true|||
-|isTemplate|Boolean|false|||
+|value|String|true|maxlength: 65535|The path to the dynamic image stored on HU or the static binary image itself.<br>Note: There is no guarantee that the image reference is valid at the time it is received.|
```
Comment (ShobhitAd) [inline docs/Common/Structs/index.md]:
```suggestion
|value|String|true|maxlength: 65535|The path to the dynamic image stored on HU or the static binary image itself.<br>**Note:** There is no guarantee that the image reference is valid at the time it is received.|
```

File: docs/Common/Structs/index.md [hunk 2]
```diff
@@ -427,9 +427,9 @@
...
+|imageType|Common.ImageType|true||Describes whether it is a static or dynamic image.|
```
Comment (ShobhitAd) [inline docs/Common/Structs/index.md]:
```suggestion
|imageType|Common.ImageType|true||Describes whether it is a `STATIC` or `DYNAMIC` image.|
```

File: docs/Common/Structs/index.md [hunk 3]
```diff
@@ -427,9 +427,9 @@
...
+|isTemplate|Boolean|false||Optional value to specify whether it's a template image. A template image can be (re)colored by the HMI as needed by using an image pattern.|
```
Comment (ShobhitAd) [inline docs/Common/Structs/index.md]:
```suggestion
|isTemplate|Boolean|false||Optional value to specify whether it's a template image.<br>**Note:** A template image can be (re)colored by the HMI as needed by using an image pattern.|
```

Comment (jacobkeeler) [reply docs/Common/Structs/index.md]: Actually, I don't think this really fits as a note, it is just defining a template image in place. Plus, this is directly pulled from the MOBILE API.

File: docs/Common/Structs/index.md (hunk 2 above)
Comment (jacobkeeler) [reply docs/Common/Structs/index.md]: Probably will revert this change too, as it appears rather odd on the actual developer portal

**Pull request merged**